.travis.yml

```diff
@@ -23,6 +29,11 @@ before_install:
         sudo apt install build-essential libgtkmm-3.0-dev xvfb
         sudo apt install libgtksourceviewmm-3.0-dev libxml++2.6-dev libsqlite3-dev
         sudo apt install libcpputest-dev gettext python3-lxml libgspell-1-dev libcurl4-openssl-dev
+        
+        sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+        sudo apt update
+        sudo apt install gcc-9 g++-9 -y
+        sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 60 --slave /usr/bin/g++ g++ /usr/bin/g++-9 
```

Comment:
Why is this necessary? Is this only for Travis or also for any bionic Ubuntu build?

Reply:
Any bionic build. The problem is that the gcc which ships with ubuntu 18 does not support c++17, I havn't used it in a while but I would guess its gcc6.

Reply:
I didn't mention it before since 1. I forgot and 2. cherrytree needs a C++17 compiler anyway